pkg/volume/azure_dd/azure_common.go

```diff
@@ -39,7 +39,7 @@ import (
 
 const (
 	defaultStorageAccountType       = storage.StandardLRS
-	defaultAzureDiskKind            = v1.AzureManagedDisk
+	defaultAzureDiskKind            = v1.AzureSharedBlobDisk
```

Comment:
regardless of the current behavior, should we now default to managed disk instead?

Reply:
Moved back to  Azure Managed.